src/doc/tarpl/subtyping.md

```diff
@@ -0,0 +1,208 @@
+% Subtyping and Variance
+
+Although Rust doesn't have any notion of inheritance, it *does* include
```

Comment:
Well, any notion of inheritance apart from traits inheriting from other traits... maybe rephrase as "doesn't have structural inheritance" or "data inheritance" ?